core-bundle/src/Resources/config/services.yml

```diff
@@ -335,6 +335,20 @@ services:
             - '@?contao.image.resize_calculator'
         public: true
 
+    Contao\CoreBundle\Mailer\AvailableTransports:
+        arguments:
+            - '@?translator'
+        tags:
+            - { name: contao.callback, table: tl_page, target: fields.mailer_transport.options, method: getTransportOptions }
+            - { name: contao.callback, table: tl_form, target: fields.mailer_transport.options, method: getTransportOptions }
```

Comment:
Why did you not use auto-configuration here?

Reply:
I thought that would not be possible with annotations, but I was wrong. Done.